assets/scss/_footer.scss

```diff
@@ -0,0 +1,16 @@
+.footer {
+    background: url("../images/footerbg.svg") center/cover no-repeat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
hmmm I looked at it and something is wrong on mobile is shows also horizontal scroll which is not good, when you remove flex it seems it's working fine
I guess you wanted to have all children centered, for the img you can use freely text-aling (it's not only for texts) and use flex just for `.footer__content`:) 
if you want to have a control for all the things inside footer you can just wrap them in another div (not that one with background with svg)
![Screenshot from 2020-04-01 00-25-03](https://user-images.githubusercontent.com/25872895/78080889-7e150800-73af-11ea-9800-f9e73f7dc40e.png)